src/pages/home/Home.js

```diff
@@ -0,0 +1,78 @@
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
+
+import './Home.css';
+
+import Post from '../../components/post/Post';
+import stringLimiter from '../../helpers/stringLimiter';
+import thousandsSeparator from '../../helpers/thousandsSeparator';
+
+const Home = () => {
+    document.title = 'Hottest posts'
+
+    const [posts, setPosts] = useState();
```

Comment:
Als `posts` eigenlijk altijd een array is, dan zou ik de default waarde aanpassen naar een lege array, dan kun je weer een extra check onderin weghalen
```suggestion
    const [posts, setPosts] = useState([]);
```